test/test_ops.py

```diff
@@ -227,17 +227,26 @@ def _ref_test_helper(self, ctx, device, dtype, op, skip_zero_numel=False, skip_z
                 )
             ):
                 continue
+
+            # Note: this deliberately uses 'op.torch_opinfo' in both cases
```

Comment:
I think this comment could be a little clearer -- what's going on in this block, and what "cases" are being described?